Xamarin.Essentials/Sms/Sms.android.cs

```diff
@@ -53,9 +53,9 @@ static Intent CreateIntent(string recipient, string body = null)
             if (!string.IsNullOrWhiteSpace(body))
                 intent.PutExtra("sms_body", body);
 
-            intent.PutExtra("address", recipient);
+            var recipienturi = string.Join(";", recipients.Select(r => AndroidUri.Encode(r)));
```

Comment:
can we make seperator of ; a const that we use

Reply:
Done